### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by vinkla**
This pull request fixes all broken links found by the awesome-bot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,31 +30,36 @@ Inspired by [ziadoz/awesome-php](https://github.com/ziadoz/awesome-php)
...
-* [Laravel API Reference](http://laravel.com/api/master)
-* [Lumen](http://lumen.laravel.com)
-* [Lumen Documentation](http://lumen.laravel.com/docs)
-* [Laracasts](http://laracasts.com)
+
+* [Laravel](https://laravel.com)
+* [Laravel Documentation](https://laravel.com/docs/5.3)
+* [Laravel API Reference](https://laravel.com/api/master/)
+* [Lumen](https://lumen.laravel.com)
+* [Lumen Documentation](https://lumen.laravel.com/docs/5.2)
```
Comment (drbyte) [inline README.md]: These version-specific (ie: 5.2, 5.3) suffixes are not needed, and should be avoided here, to save having to update every time a new release is issued.

File: README.md [hunk 2]
```diff
@@ -163,103 +174,109 @@ Inspired by [ziadoz/awesome-php](https://github.com/ziadoz/awesome-php)
...
 
 ## Development Setup
-* [Homestead](http://laravel.com/docs/homestead) - Official Vagrant box for Laravel
+
+* [Homestead](https://laravel.com/docs/5.3/homestead) - Official Vagrant box for Laravel
   * [Getting Started with Laravel Homestead](https://scotch.io/tutorials/getting-started-with-laravel-homestead)
   * [Installation on OSx and Linux](https://laracasts.com/series/laravel-5-from-scratch/episodes/3)
   * [Installation on  Windows](http://blog.teamtreehouse.com/laravel-homestead-on-windows)
-* [Valet](http://laravel.com/docs/valet) - development environment for Mac minimalists
+* [Valet](https://laravel.com/docs/5.3/valet/) - development environment for Mac minimalists
```
Comment (drbyte) [inline README.md]: Again, version number should be avoided here

Comment (vinkla) [inline README.md]: Maybe, the tests will fail though. Lets keep it like this for now and update it in the future. We're only required to update the version number once or twice a year.

File: README.md (hunk 1 above)
Comment (vinkla) [inline README.md]: Lets keep it like this for now and update it in the future. We're only required to update the version number once or twice a year.

**Comment by vinkla** [comment]
@chiraggude this pull request is now ready to merge.

**Comment by chiraggude** [comment]
Not sure about the version specific links to docs. It's fine for now but I might change them later.

**Comment by vinkla** [comment]
Sounds good!

**Pull request merged**